channel-api-gb.env

```diff
@@ -1,12 +1,43 @@
+# CHANNEL API CONFIG SECTION
+# ---------------------------------------------
+# port to which channel api binds
 PORT=9091
-
-SENDER_REF=GB
-
-# AWS_REGION=us-east-1
-
-CONTRACT_BUCKET=contract
+# used to retrieve ChannelNode contract interface and address
+IGL_CONTRACT_REPO_BUCKET=contract
 CONTRACT_BUILD_ARTIFACT_KEY=channel-node-participant-gb/ChannelNode.json
 CONTRACT_NETWORK_ID=15
-CONTRACT_OWNER_PRIVATE_KEY=0x4f3edf983ac636a65a842ce7c78d9aa706d3b113bce9c46f30d7d21715b23b1d
-
+# used to sign transactions of ChannelNode contract method calls
+CONTRACT_OWNER_PRIVATE_KEY=0x6cbed15c793ce57650b9877cf6fa156fbef513c4e6134f022a85b1ffdd59b2a1
+# used to determine message delivery status
+MESSAGE_CONFIRMATION_THRESHOLD=12
+# used to set the respective message parameters
+SENDER=GB
+SENDER_REF=sender_ref:GB
+# used to establish blockchain connection
 HTTP_BLOCKCHAIN_ENDPOINT=http://tec-ganache-cli:8585
+# used to store websub subscriptions data
+IGL_SUBSCRIPTIONS_REPO_BUCKET=subscriptions-gb
+# used to verify topic canonical urls and for setting websub notifications Link headers.
+CHANNEL_URL=http://tec-channel-api-gb:9091/
+# ---------------------------------------------
+
+# NEW MESSAGE OBSERVER PROCESSOR CONFIG SECTION
+# ---------------------------------------------
+IGL_NOTIFICATIONS_REPO_QNAME=notifications-gb
+IGL_CHANNEL_REPO_QNAME=channel-gb
+RECEIVER=GB
+# ---------------------------------------------
+
+# CALLBACK SPREADER PROCESSOR CONFIG SECTION
+# ---------------------------------------------
+IGL_NOTIFICATIONS_REPO_QNAME=notifications-gb
```

Comment:
it's common to use SG as the second setup

Reply:
Ye, I understood that too far into the development process to change this quickly. But, anyway, that doesn't hurt too much, probably I'll change it later.